ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -1827,7 +1810,10 @@ module AuthADWinbind : Auth_signature.AUTH_MODULE = struct
       ClosestKdc.trigger_update ~start:0. ;
       RotateMachinePassword.trigger_rotate ~start:0. ;
       ConfigHosts.join ~domain:service_name ~name:netbios_name ;
-      ResolveConfig.join ~domain:service_name ;
+      Forkhelpers.execute_command_get_output !Xapi_globs.set_hostname
```

Comment:
Minof: it is better to be explicit about the return type to avoid accidental partial applications that would raise a compiler warning and not call anything at runtime.
Suggestion:
```
let (_, _) = Forkhelpers.execute_command_get_output !Xapi_globs.set_hostname
                    [get_localhost_name ()]
```